<ui version="4.0" >
 <class>Qt5</class>
 <widget class="QMainWindow" name="Qt5" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLabel" name="lb1" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>200</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionCopy" />
    <addaction name="actionPast" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="separator" />
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="text" >
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPast" >
   <property name="text" >
    <string>Past</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
